cli/src/cmd_docs.rs

```diff
@@ -66,6 +66,7 @@ fn to_json(cmd: &Command) -> JsonDoc {
     let mut args = cmd
         .get_arguments()
         .filter(|arg| arg.get_long() != Some("help"))
+        .filter(|arg| !arg.is_hide_set())
```

Comment:
this is fine, but is it still relevant?

Reply:
I think it's worth leaving in to stop documenting the [--parallelism argument](https://github.com/oxidecomputer/oxide.rs/commit/52d739ace616f2669bfb07be60dce7cfa83bf9df#diff-d79b1422a7679e4c16d6d959d8f93c039eeac3de0a28fb3b18c26198e47dae26L1054)